ios/Sources/TimetableFeature/List/TimetableLoaded.swift

```diff
@@ -149,14 +171,17 @@ private extension SelectedType {
 }
 private extension TimetableDetailScreen.ViewState {
     init?(state: TimetableLoadedState) {
-        guard let detail = state.detail else { return nil }
-        timetable = detail
+        guard let selectedTimetable = state.selectedTimetable else { return nil }
+        timetable = selectedTimetable
     }
 }
 
 private extension TimetableLoadedAction {
     init(action: TimetableDetailScreen.ViewAction) {
-        self = .none
+        switch action {
+        case .tapLink(let link):
+            self = .tapLink(link)
+        }
     }
 }
```

Comment:
These action and state should be placed in TimetableDetail** not ViewState and ViewAction.